web/src/immutant/web/undertow.clj

```diff
@@ -109,10 +109,12 @@
   "Assoc an SSLContext given a keystore and a truststore, which may be
   either actual KeyStore instances, or paths to them. If truststore is
   ommitted, the keystore is assumed to fulfill both roles"
-  [{:keys [keystore key-password truststore trust-password] :as options}]
-  (-> options
-    (assoc :ssl-context (keystore->ssl-context options))
-    (dissoc :keystore :key-password :truststore :trust-password)))
+  [{:keys [keystore key-password truststore trust-password ssl-context] :as options}]
```

Comment:
oops, my bad

Reply:
I don't see any "bads" on this line :) `ssl-context` wasn't used in the original code, so there was no reason to bind it.

Reply:
This code is totally bonkers, none of these bindings are used, but are implicitly relied upon by `valid-options` on line 172 below.

Reply:
Oh, huh! Neat. Fun with introspection.

I guess the initial vector on 172 could be edited to remove `#'ssl-context` now that we've added this.